conftest.py

```diff
@@ -147,30 +192,55 @@ def setup_artifact_path(item: pytest.Item, artifact_name: str):
     return os.path.join(output_dir, artifact_name)
 
 
-def get_rotating_task_log_lines(task_id: str, known_task_files: set, task_file: str):
+def get_task_files_for_id(task_id: str):
```

Comment:
Could we annotate that this returns a `set`?
```python
def get_task_files_for_id(task_id: str) -> set:
```

Reply:
Done.